maintenance/tables.sql

```diff
@@ -967,7 +967,8 @@ CREATE TABLE /*_*/recentchanges (
 
   -- As in revision
   rc_user int unsigned NOT NULL default 0,
-  rc_user_text varchar(255) binary NOT NULL,
+  -- SUS-3079: Placeholder empty value until migration
+  rc_user_text VARCHAR(255) BINARY NOT NULL DEFAULT '',
```

Comment:
Do we need to update the schema site-wide before we move the next step?

Reply:
It'll be handled in `update.php`. Post-merge, it will migrate data to `rc_ip_bin` column, drop indexes, and add this default value to already created `rc_user_text` columns. Then we can remove explicit value setting for `rc_user_text` in inserts, and drop the column entirely.